apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -59,6 +67,24 @@ const { RESIDENT } = require('@condo/domains/user/constants/common')
 const { classifyTicket } = require('../utils/serverSchema/resolveHelpers')
 
 
+const ERRORS = {
+    FEEDBACK_VALUE_MUST_BE_SPECIFIED: {
+        code: BAD_USER_INPUT,
+        type: WRONG_VALUE,
+        message: '"feedbackValue" must be specified if there is "feedbackAdditionalOptions" or "feedbackComment"',
+        messageForUser: 'api.ticket.FEEDBACK_VALUE_MUST_BE_SPECIFIED',
+    },
+}
+
+// todo(doma-5776): should delete this function when will done task DOMA-5319
+const convertTextOptionToKeyOption = (option) => {
+    if (option === 'Сделали на совесть' || option === 'Made in good faith') return FEEDBACK_ADDITIONAL_OPTIONS_BY_KEY.HIGH_QUALITY
```

Comment:
Hardcode! You shold get some const value inside API. And do this convert on the front-end side